company-profiles/basisPointDesignLtd.md

```diff
@@ -0,0 +1,25 @@
+# basispointdesignltd
```

Comment:
The company name and file name should match the name listed in the readme.md file. This company is listed as `BasisPointDesign` so the file name should be `basispointdesign.md` and the title here should be `# BasisPointDesign`